CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ instructions below to upgrade your Postgres.
 * [CHANGE] Overrides mechanism has been renamed to "runtime config", and is now separate from limits. Runtime config is simply a file that is reloaded by Cortex every couple of seconds. Limits and now also multi KV use this mechanism.<br />New arguments were introduced: `-runtime-config.file` (defaults to empty) and `-runtime-config.reload-period` (defaults to 10 seconds), which replace previously used `-limits.per-user-override-config` and `-limits.per-user-override-period` options. Old options are still used if `-runtime-config.file` is not specified. This change is also reflected in YAML configuration, where old `limits.per_tenant_override_config` and `limits.per_tenant_override_period` fields are replaced with `runtime_config.file` and `runtime_config.period` respectively. #1749
 * [CHANGE] Cortex now rejects data with duplicate labels. Previously, such data was accepted, with duplicate labels removed with only one value left. #1964
 * [CHANGE] Changed the default value for `-distributor.ha-tracker.prefix` from `collectors/` to `ha-tracker/` in order to not clash with other keys (ie. ring) stored in the same key-value store. #1940
+* [CHANGE] Moved `store-min-chunk-age` to the Querier config as `querier.query-store-after`, allowing the store to be skipped during query time if the metrics wouldn't be found. IngesterMaxQueryLookback as been renamed to QueryIngestersWithin to make it match its flag and yaml settings.#1893 
```

Comment:
`store-min-chunk-age` > `store.min-chunk-age` and please prefix flags with `-` to make it more clear they're CLI flags.

Reply:
Fixed.